app/src/app/components/lia-toast/lia-toast.component.ts

```diff
@@ -0,0 +1,49 @@
+import { Component, Input, Output, EventEmitter, OnChanges, ChangeDetectionStrategy, ViewEncapsulation } from '@angular/core';
+import { NgClass } from '@angular/common';
+
+@Component({
+  moduleId: module.id,
+  selector: 'lia-toast',
+  templateUrl: 'lia-toast.component.html',
+  styleUrls: ['lia-toast.component.css'],
+  encapsulation: ViewEncapsulation.None,
+  directives: [NgClass]
+})
+
+export class LiaToastComponent implements OnChanges {
+  @Input() msg: string;
+  @Output() onChangeMsg = new EventEmitter();
+
+  classNames: any;
+
+  constructor() {
+    this.classNames = {
+      'lia-toast': true,
+      'open': false
+    };
+  }
+
+  ngOnChanges(changes: any) {
+    if ('msg' in changes && changes.msg.currentValue) {
+      setTimeout(() => {
+        this.classNames.open = true;
+      }, 10);
```

Comment:
I am not sure I understand why we have such a small value for `setTimeout` and what was the intention ?